src/app/Core/ExportServices/generateCSVString.service.ts

```diff
@@ -14,14 +14,19 @@ function stringModification(inputString: string): string {
  * @returns {string} - data in one csv string
  */
 export function generateCSVString<
-    T extends Record<string, string>,
+    T extends Record<string, unknown>,

```

Comment:
why is this unknown? Your code should be aware of how the data looks like it processes

Reply:
Already changed it in the commit: "Adaption to isolate API "

Reply:
then your typechecking code below is also obsolete?

Reply:
done. 
- Remove TypeCheck in generateCSVString